src/distilabel/llm/anthropic.py

```diff
@@ -52,14 +55,20 @@ class AnthropicLLM(AsyncLLM):
         max_retries: the maximum number of retries for the LLM. Defaults to 2.
     """
 
-    model: str = "claude-3-opus-20240229"
-    api_key: Optional[SecretStr] = os.getenv("ANTHROPIC_API_KEY", None)  # type: ignore
-    base_url: Union[str, None] = None
+    model: str
+    base_url: Optional[RuntimeParameter[str]] = Field(
+        default=os.getenv("ANTHROPIC_BASE_URL", "https://api.anthropic.com"),
```

Comment:
With this approach the value for `ANTHROPIC_BASE_URL` won't be updated unless set before import or reloading the imports, and IMO that's not really user friendly, as you may be assigning and environment variable value within the code as i.e. `os.environ["ANTHROPIC_BASE_URL"]` but that won't be applied, and indeed only used if reloading the import :/ Is there any way to always init i.e. call the `Field` intializer when instantiating a `pydantic.BaseModel` via `model_config`?

Reply:
Yes, I agree, I wasn't conviced either. I'll change the approach.